src/Microsoft.ML.ImageAnalytics/ImageBase.cs

```diff
@@ -0,0 +1,480 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using SkiaSharp;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.ML.Data
```

Comment:
@michaelgsharp @ericstj @luisquintanilla I put the new interfaces inside  `Microsoft.ML.Data`. Do you think it is better to put these inside more generic names like `Microsoft.ML` or `Microsoft.ML.Imaging`, or whatever else?

Reply:
If we want others to be able to take a dependency on them (like torchsharp or something) we should have them in a different package. If not I think there is a good place for them as most other types live there.

Reply:
Thanks @michaelgsharp, we decided we'll expose the new APIs scoped to ML.NET. the last commit in this PR showing the change.